packages/mongodb-chatbot-ui/src/FloatingActionButtonTrigger.tsx

```diff
@@ -1,5 +1,4 @@
-import { css } from "@emotion/css";
-import classNames from "classnames";
+import { css, cx } from "@emotion/css";
```

Comment:
why these changes? can you note why in the PR description?

Reply:
Basically just tidying up - we don't need this lib because emotion has their own version baked in